### ISSUE TITLE
How to set group_size > 1

### ISSUE THREAD

**Issue by vdv1g16** [tensorflow/ranking#63]
Whenever group_size > 1, I got the following error:
InvalidArgumentError (see above for traceback): Input to reshape is a tensor with 200 values, but the requested shape has 400
	 [[node groupwise_dnn_v2/accumulate_scores/Reshape (defined at /home/vdv1g16/Desktop/Learning to rank/tf-idf/ranking/tensorflow_ranking/python/model.py:245) ]]

Here the requested shape is 400 ( when group_size = 2 ) and 600 ( when group_size = 3) .

I guess the error comes from the model.py:
scores = _score_fn(large_batch_context_features, large_batch_group_features, reuse=False)      with tf.name_scope('accumulate_scores'):
     scores = tf.reshape(scores, [batch_size, num_groups, group_size])

In the tf_ranking_libsvm.ipynb, when I change the dimension of output, the code run well.Please explain how to do it in a right way 

def make_score_fn():
  """Returns a scoring function to build `EstimatorSpec`."""
  def _score_fn(context_features, group_features, mode, params, config):
    """Defines the network to score a documents."""
    del params
    del config
    # Define input layer.
    example_input = [tf.layers.flatten(group_features[name])
                    for name in sorted(example_feature_columns())]
    input_layer = tf.concat(example_input, 1)
    cur_layer = input_layer
    for i, layer_width in enumerate(int(d) for d in _HIDDEN_LAYER_DIMS):
      cur_layer = tf.layers.dense(
          cur_layer,
          units=layer_width,
          activation="tanh")
    logits = tf.layers.dense(cur_layer, units=20)
    print(cur_layer)
    return logits
  return _score_fn

**Comment by ramakumar1729** [comment]
This error could happen if you have an existing checkpoint for group_size = 1, and then try to run training for group_size = 2. This can also happen if you train for group size = 1, but try to run evaluation with group_size = 2. Can you check if either of these two cases are relevant for your problem?

**Comment by vdv1g16** [comment]
I run the tf_ranking_libsvm.ipynb file and I only adjust the group_size to have the listwise score. I do not think that I have an existing checkpoint (if yes How do I find the path). I only use the tf_ranking_libsvm.ipynb   with your data.

**Comment by xuanhuiwang** [comment]
Can you try the script version tf_ranking_libsvm.py?

**Comment by whatsmyname** [comment]
it works. THANKS.
